json/aerospike/6.4.0.json

```diff
@@ -0,0 +1,2881 @@
+{
+  "$schema": "http://json-schema.org/draft-06/schema",
+  "additionalProperties": false,
+  "type": "object",
+  "required": ["network", "namespaces"],
+  "properties": {
+    "service": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "advertise-ipv6": {
+          "type": "boolean",
+          "default": false,
+          "description": "",
+          "dynamic": true
+        },
+        "auto-pin": {
+          "type": "string",
+          "description": "",
+          "dynamic": false,
+          "default": "none",
+          "enum": ["none", "cpu", "numa", "adq"]
+        },
+        "batch-index-threads": {
+          "type": "integer",
+          "default": 1,
+          "minimum": 1,
+          "maximum": 256,
+          "description": "",
+          "dynamic": true
+        },
+        "batch-max-buffers-per-queue": {
+          "type": "integer",
+          "default": 255,
+          "minimum": 0,
+          "maximum": 4294967295,
+          "description": "",
+          "dynamic": true
+        },
+        "batch-max-unused-buffers": {
+          "type": "integer",
+          "default": 256,
+          "minimum": 0,
+          "maximum": 4294967295,
+          "description": "",
+          "dynamic": true
+        },
+        "cluster-name": {
+          "type": "string",
+          "default": "",
+          "description": "",
+          "dynamic": true
+        },
+        "debug-allocations": {
+          "type": "string",
+          "description": "",
+          "dynamic": false,
+          "default": "none",
+          "enum": ["none", "transient", "persistent", "all"]
+        },
+        "disable-udf-execution": {
+          "type": "boolean",
+          "default": false,
+          "description": "",
+          "dynamic": false
+        },
+        "enable-benchmarks-fabric": {
+          "type": "boolean",
+          "default": false,
+          "description": "",
+          "dynamic": true
+        },
+        "enable-health-check": {
+          "type": "boolean",
+          "default": false,
+          "description": "",
+          "dynamic": true
+        },
+        "enable-hist-info": {
+          "type": "boolean",
+          "default": false,
+          "description": "",
+          "dynamic": true
+        },
+        "enforce-best-practices": {
+          "type": "boolean",
+          "default": false,
+          "description": "",
+          "dynamic": false
+        },
+        "feature-key-file": {
+          "type": "string",
+          "default": "/opt/aerospike/data/features.conf",
+          "description": "",
+          "dynamic": false
+        },
+        "feature-key-files": {
```

Comment:
We have removed the required validation for these fields. Are we going to do the same for older version schemas as well?

Reply:
This validation was already removed from previous versions in an old PR